docs/workflow/building/libraries/code-coverage.md

```diff
@@ -34,17 +34,17 @@ This runs the tests and generates the full code coverage report. The resulting i
 
     artifacts\coverage\index.htm
 
-You can also build and test with code coverage for a particular test project rather than for the whole repo with the `/p:Coverage=true` property:
+You can also build and test with code coverage for a particular test project rather than for the whole repo with the `--collect:"XPlat Code Coverage"` switch:
 
-    dotnet build /t:Test /p:Coverage=true
+    dotnet test --collect:"XPlat Code Coverage"
```

Comment:
can we somehow preserve the `/p:Coverage=true` flag?

Reply:
Currently not, but I worked on https://github.com/microsoft/vstest/pull/2429 a few months ago which would enable that. Just hadn't had time to finish it.